staging/src/k8s.io/component-helpers/storage/volume/pv_helpers.go

```diff
@@ -225,19 +222,17 @@ func FindMatchingVolume(
 		}
 
 		// check if PV's DeletionTimeStamp is set, if so, skip this volume.
-		if utilfeature.DefaultFeatureGate.Enabled(features.StorageObjectInUseProtection) {
-			if volume.ObjectMeta.DeletionTimestamp != nil {
-				continue
-			}
+		if enableStorageObjectInUseProtection && volume.ObjectMeta.DeletionTimestamp != nil {
```

Comment:
Should be cleaned up in a separate PR, but this feature has been GA since 1.11 and we can remove the feature gate.

Reply:
oh, let's do that first